### ISSUE TITLE
[DOCKER] Jellyfish is unable to load the shared library

### ISSUE THREAD

**Issue by greatfireball** [nterhoeven/reper#5]
After building the image, jellyfish is not able to load its shared library.

```
root@1bee6e683f33:/data# jellyfish --version
jellyfish: error while loading shared libraries: libjellyfish-2.0.so.2: cannot open shared object file: No such file or directory
```

**Comment by greatfireball** [comment]
I am working on the fix of that issue and it will be fixed via pull request

**Comment by nterhoeven** [comment]
I merged you pull request #7 - it works, I am closing this issue.
